Docs/source/developers/qed.rst

```diff
@@ -40,6 +40,6 @@ pairs created.
 At most one macroparticle is created per cell per timestep per species, with a weight corresponding to
 the total number of physical pairs created.
 
-So far the Schwinger module requires using ``warpx.do_nodal=1`` or
+So far the Schwinger module requires using ``warpx.do_centered = 1`` or
 ``algo.field_gathering=momentum-conserving`` (so that the auxiliary fields are calculated on the nodes)
```

Comment:
@lucafedeli88 @NeilZaim Do we have a CI test that uses the Schwinger module with `warpx.do_nodal = 1` (which would become `warpx.do_centered = 1` in this PR)? If not, we might be breaking the validity of the code here (by switching from fully nodal to fully cell-centered) without noticing it.

Reply:
Hi @EZoni and sorry for the late reply.

The Schwinger CI test uses the momentum conserving option so this PR should not affect it (fields are still on the nodes).

However, this PR might very slightly affect the Schwinger module itself: with the `do_centered` option, the particles would still be created on the nodes while the fields would be evaluated on the cell centers. I don't think that this would be a very significant issue so it's probably fine if this fixed in a follow-up PR (which I can take care of once this PR is merged).